src/logging.cpp

```diff
@@ -45,7 +45,7 @@ long long SimpleLogger::maxPayloadLogSize  = 10240;
 #ifdef ENABLE_LOG_PERFORMANCE
 
 #ifdef WIN32
-thread_local std::array<char, LOGGER_CHUNKS_SIZE> SimpleLogger::mBuffer; 
+//thread_local std::array<char, LOGGER_CHUNKS_SIZE> SimpleLogger::mBuffer; 
```

Comment:
```suggestion
#ifdef _WIN64
    thread_local std::array<char, LOGGER_CHUNKS_SIZE> SimpleLogger::mBuffer; // Keep this as a normal class member on WIN32 until we abandon XP (Qt fonts appear with strikethrough on XP otherwise)
#endif
```